samples/test/transcoder.test.js

```diff
@@ -43,6 +43,14 @@ const outputUriForTemplate = `gs://${bucketName}/test-output-template/`;
 const outputUriForAdHoc = `gs://${bucketName}/test-output-adhoc/`;
 const outputUriForStaticOverlay = `gs://${bucketName}/test-output-static-overlay/`;
 const outputUriForAnimatedOverlay = `gs://${bucketName}/test-output-animated-overlay/`;
+const outputDirForSetNumberImagesSpritesheet =
+  'test-output-set-number-spritesheet/';
+const outputUriForSetNumberImagesSpritesheet = `gs://${bucketName}/${outputDirForSetNumberImagesSpritesheet}`;
+const outputDirForPeriodicImagesSpritesheet =
+  'test-output-periodic-spritesheet/';
+const outputUriForPeriodicImagesSpritesheet = `gs://${bucketName}/${outputDirForPeriodicImagesSpritesheet}`;
+const smallSpriteSheetFileName = 'small-sprite-sheet0000000000.jpeg';
```

Comment:
Nit: perhaps put a comment as to where these resources can be found? If the images are relatively small, it also might be worthwhile to add a `samples/resources` folder with them?